pkg/controller/v1beta1/inferenceservice/rawkube_controller_test.go

```diff
@@ -9278,13 +9278,13 @@ var _ = Describe("v1beta1 inference service controller", func() {
 					},
 				},
 				URL: &apis.URL{
-					Scheme: "http",
+					Scheme: "https",
```

Comment:
Can we add a comment saying it should be https for ODH? To make future syncs easier